packages/react/scripts/env.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const BABEL_ENV = process.env.BABEL_ENV;
+
+module.exports = () => ({
+  presets: [
+    [
+      require.resolve('@babel/preset-env'),
+      {
+        modules: BABEL_ENV === 'es' ? false : 'commonjs',
+        targets: {
+          browsers: ['extends browserslist-config-carbon'],
```

Comment:
This looks like an unpublished carbon package, right?

Reply:
Yes, there are several unpublished packages in the project, leveraging Yarn workspaces.